x-pack/plugins/apm/server/lib/rum_client/has_rum_data.ts

```diff
@@ -31,9 +31,7 @@ export async function hasRumData({ setup }: { setup: Setup & SetupTimeRange }) {
         },
         aggs: {
           services: {
-            filter: {
-              range: rangeFilter(start, end),
-            },
+            filter: rangeQuery(start, end)[0],
```

Comment:
rangeQuery returns an array or object?

Reply:
i think it returns array, i guess it can be assigned directly here.

Reply:
These query functions return arrays because they are normally spread. This is because `filter: [{}]` is an error in Elasticsearch queries but `filter: [...[]]` is not.